charts/trento-server/charts/trento-mcp-server/values.yaml

```diff
@@ -0,0 +1,261 @@
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: Apache-2.0
+
+## @section Common parameters
+##
+## @param image.repository Image repository
+## @param image.tag Image tag
+## @param image.pullPolicy Image pull policy
+image:
+  repository: ghcr.io/trento-project/trento-mcp-server
+  tag: latest
+  pullPolicy: Always
+
+## @param replicaCount Number of replicas to deploy
+replicaCount: 1
+
+## @param containerPorts.http Port for MCP HTTP traffic
+## @param containerPorts.health Port for health check
+containerPorts:
+  http: 5000
+  health: 8080
+
+## @section MCP Server configuration
+##
+## @param mcpServer.args Array of arguments to pass to the MCP server container, overrides other settings
+## @param mcpServer.autodiscoveryPaths Custom paths for API autodiscovery
+## @param mcpServer.enableHealthCheck Enable health check endpoint
+## @param mcpServer.headerName Name of the header the MCP client should use to pass the Trento Personal Access Token
+## @param mcpServer.healthApiPath The API path used for health checks on target servers, like Trento Web or Wanda
+## @param mcpServer.insecureTLS Disable TLS certificate verification
+## @param mcpServer.oasPath List of paths to OpenAPI specification files. If empty, it defaults to Trento Web and Wanda internal services
+## @param mcpServer.tagFilter List of tags to filter
+## @param mcpServer.transport Transport protocol for the server
+## @param mcpServer.trentoURL URL of the Trento server. If empty, it defaults to Trento Web internal service
+## @param mcpServer.verbosity Log level verbosity
+mcpServer:
+  args: []
+  autodiscoveryPaths:
+    - /api/all/openapi
+    - /wanda/api/all/openapi
+  enableHealthCheck: true
+  headerName: Authorization
```

Comment:
I've changed it to `Authorization`. It has no other implications, nor is it related to any Trento headers.
The reason is that OpenWebUI does not support custom header names yet... so setting this default value should improve the installation UX, I think.